webrender/src/device/gfx/device.rs

```diff
@@ -3164,6 +3164,12 @@ impl<B: hal::Backend> Device<B> {
                 color_clear.into_iter().chain(depth_clear.into_iter()),
                 hal::command::SubpassContents::Inline,
             );
+            self.command_buffer.set_viewports(0, &[self.viewport.clone()]);
+            if self.scissor_enabled {
```

Comment:
nit: could do `let rect = if ... {.. } else {...};` and then have `self.command_buffer.set_scissors` only once

Reply:
Fixed